File: cash++/cash.c
#include <cs50.h>
#include <stdio.h>
 #include <math.h>

    
    
  int count(void){
int counter=0;
    float cash=0;
    int quarter = 25;
    int dime = 10;
    int nickel = 5;
    int penny = 1;
  
    do{
        cash=get_float("Change owed: ");
    }
    
      while(cash<0);
    
    int cents = round(cash * 100);
    while(cents!=0)
    {
        if(cents/quarter!=0)
        {
          cents=cents-quarter;
              counter++;
        }
        else if(cents/dime!=0)
        {
          cents=cents-dime;
              counter++;
        }
        else if(cents/nickel!=0)
        {
          cents=cents-nickel;
              counter++;
        }
        else
        {
          cents=cents-penny;
              counter++;  
        }
    }
  
   return counter;
  }
int main(void)
{
        printf("%i\n", count());

}

File: cashPy/cash.py
from cs50 import get_float

conies=get_float("Change owed:")

while conies < 0:

    coines = get_float("Change owed:")
count = 0

conies = round(conies*100)


while conies != 0:
    if conies // 25 != 0:
       conies =conies-25
       count=count+1
    elif conies // 10 != 0:
       conies =conies-10
       count=count+1

    elif conies // 5 != 0:
       conies =conies-5
       count=count+1
    else:
        conies =conies-1
        count=count+1
print(count)



















# elif conies / 10 !=0:
    #      conies =conies-10
    #      count+1
    # elif conies / 5 !=0:
    #      conies =conies-5
    #      count+1
    # elif conies / 1 !=0:
    #      conies =conies-1
    #      count+1